munimap/config.py

```diff
@@ -205,3 +204,5 @@ class TestConfig(object):
     PRINT_USE_BROKER = False
     PRINT_QUEUEFILE = path.join(here, 'test/data/printqueue.sqlite')
     WTF_CSRF_ENABLED = False
+
+    LOG_DIR = path.join(here, 'test/logs/')
```

Comment:
I am currently working on a separate branch to fix and update the test packages. Since this is related to the test update that i started, I will remove it from here.

Reply:
Seems like you forgot to actually remove it :smiley:

Reply:
Looks like this is still here